pages/static/src/scss/components/_nav.scss

```diff
@@ -561,47 +528,51 @@
     background: darken(#f5f5f5, 10);
   }
 }
-// 
-// @media (min-width: 768px) {
-// 
-//   .navbar-brand {
-//     padding: 15px;
-//   }
-// 
-//   .navbar .logo {
-//     height: 50px;
-//     width: auto;
-//     transition: 0.25s;
-// 
-//     &:hover {
-//       transform: scale(1.1);
-//     }
-//   }
-// 
-//   .navbar {
-//     height: 80px;
-//   }
-// 
-//   .navbar-default .navbar-nav > li > a {
-//     padding-top: 30px;
-//     padding-bottom: 30px;
-//   }
-// 
-//   .navbar .btn-donate.navbar-btn {
-//     margin-top: 17px;
-//     margin-bottom: 17px;
-//     padding: 11px 18px;
-//   }
-// 
-//   .navbar-nav--centered {
-//     text-align: center;
-//     position: absolute;
-//     left: 0;
-//     right: 0;
-// 
-//     > li {
-//       float: none;
-//       display: inline-block;
-//     }
-//   }
-// }
+
+.navbar--microsite {
```

Comment:
Suggest moving these to microsite scss file? That might be better organization, not sure.

Reply:
I thought about that, not sure what made the most sense. Probably should move to microsite.

Reply:
I think if possible to keep all them microsite styles together that might be best. I forget if there is a good way to flag dependencies in scss files.